latex_template/template.tex

```diff
@@ -22,3 +22,8 @@
 \newcommand{\examhead}[3]{\section{#1 Paper #2 Question #3}}
 \newenvironment{examquestion}[3]{\examhead{#1}{#2}{#3}\setlist[enumerate, 1]{label=(\alph*)}\setlist[enumerate, 2]{label=(\roman*)}}{}
 
+\documentclass[10pt,twoside,a4paper]{article}
```

Comment:
Shouldn't this be the first line of template.tex?

Reply:
Ah yes, sorry, I'll do that now.